src/pages/components/MinuteSelector.tsx

```diff
@@ -1,23 +1,36 @@
 import React from 'react'
 import { TEXTS } from 'src/resources/texts'
 import { TextField } from '@mui/material'
+import ClearButton from './ClearButton'
 
 type MinuteSelectorProps = {
   num: number
   setNum: (num: string) => void
 }
 
 const MinuteSelector = ({ num, setNum }: MinuteSelectorProps) => {
+  const handleClearInput = () => {
+    setNum('0')
```

Comment:
why the number is a string?

Reply:
I did as the interface required from me.
`type MinuteSelectorProps = {
  num: number;
  setNum: (num: string) => void
}`

I will change the interface to be:
`type MinuteSelectorProps = {
  num: number;
  setNum: (num: number) => void
}`

And after I will write it as a number and not string.